src/blocking.rs

```diff
@@ -251,47 +239,44 @@ impl BlockingIronOxide {
         timeout: Option<std::time::Duration>,
     ) -> Result<DeviceAddResult> {
         let rt = create_runtime();
-        rt.enter(|| {
-            block_on(IronOxide::generate_new_device(
-                jwt,
-                password,
-                device_create_options,
-                timeout,
-            ))
-        })
+        rt.block_on(IronOxide::generate_new_device(
+            jwt,
+            password,
+            device_create_options,
+            timeout,
+        ))
     }
     /// See [ironoxide::user::UserOps::user_delete_device](trait.UserOps.html#tymethod.user_delete_device)
     pub fn user_delete_device(&self, device_id: Option<&DeviceId>) -> Result<DeviceId> {
         self.runtime
-            .enter(|| block_on(self.ironoxide.user_delete_device(device_id)))
+            .block_on(self.ironoxide.user_delete_device(device_id))
     }
     /// See [ironoxide::user::UserOps::user_verify](trait.UserOps.html#tymethod.user_verify)
     pub fn user_verify(
         jwt: &Jwt,
         timeout: Option<std::time::Duration>,
     ) -> Result<Option<UserResult>> {
         let rt = create_runtime();
-        rt.enter(|| block_on(IronOxide::user_verify(jwt, timeout)))
+        rt.block_on(IronOxide::user_verify(jwt, timeout))
     }
     /// See [ironoxide::user::UserOps::user_get_public_key](trait.UserOps.html#tymethod.user_get_public_key)
     pub fn user_get_public_key(&self, users: &[UserId]) -> Result<HashMap<UserId, PublicKey>> {
         self.runtime
-            .enter(|| block_on(self.ironoxide.user_get_public_key(users)))
+            .block_on(self.ironoxide.user_get_public_key(users))
     }
     /// See [ironoxide::user::UserOps::user_rotate_private_key](trait.UserOps.html#tymethod.user_rotate_private_key)
     pub fn user_rotate_private_key(&self, password: &str) -> Result<UserUpdatePrivateKeyResult> {
         self.runtime
-            .enter(|| block_on(self.ironoxide.user_rotate_private_key(password)))
+            .block_on(self.ironoxide.user_rotate_private_key(password))
     }
 }
 
 /// Creates a tokio runtime with the default number of core threads (num of cores on a machine)
 /// and an elevated number of blocking_threads as we expect heavy concurrency to be network-bound
 fn create_runtime() -> tokio::runtime::Runtime {
-    tokio::runtime::Builder::new()
-        .threaded_scheduler() // use multi-threaded scheduler
+    tokio::runtime::Builder::new_multi_thread()
         .enable_all() // enable both I/O and time drivers
-        .max_threads(250) // core_threads default to number of cores, blocking threads are max - core
+        .max_blocking_threads(250) // worker_threads default to number of cores
```

Comment:
I'll need to do some reading on tokio 1.0's threading model, but in the past blocking threads wouldn't be what we wanted here...

Reply:
Well, this is off by the number of core threads, but I don't think that's worth arguing over. As the previous comment said setting max threads was setting blocking threads to 250-(core threads)

Reply:
I figured that we chose this number fairly arbitrarily in the first place, so it changing by a few while also being a bit more clear wouldn't be bad

Reply:
We aren't doing any spawn_blocking, so this can be removed.